### ISSUE TITLE
[Diffusers]: Diffuser generator is defaulting to CPU: DirectML does not support torch Generator API

### ISSUE THREAD

**Issue by Synitare** [vladmandic/automatic#1868]
### Issue Description

I'm getting the following error when attempting to set the diffuser back to default:

12:24:56-262019 WARNING  DirectML does not support torch Generator API.
12:24:56-265019 WARNING  diffusers_generator_device is automatically overriden to cpu.
12:24:56-266020 INFO     3 options are automatically overriden. If you want to keep them from overriding, run with
                         --experimental argument.
12:24:56-268019 INFO     Settings changed: 1 ['diffusers_generator_device']

### Diffusers pipeline used

Stable Diffusion. (Not SDXL)

### Version Platform Description

12:22:18-419720 INFO     Starting SD.Next
12:22:18-423719 INFO     Python 3.10.11 on Windows
12:22:18-460720 INFO     Version: 77de9cd0 Fri Jul 28 23:18:28 2023 +0900
12:22:18-859128 INFO     Latest published version: 9f18db474d669d7ba57233d91f68b226d13b1370 2023-07-28T17:15:31Z
12:22:18-864129 INFO     nVidia CUDA toolkit detected
12:22:18-992129 INFO     Disabled extensions: ['stable-diffusion-webui-images-browser']
12:22:18-993128 INFO     Enabled extensions-builtin: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR',
                         'Lora', 'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-dynamic-thresholding',
                         'sd-extension-system-info', 'sd-webui-agent-scheduler', 'sd-webui-controlnet',
                         'stable-diffusion-webui-rembg', 'SwinIR']
12:22:19-000128 INFO     Enabled extensions: ['a1111-sd-webui-tagcomplete', 'adetailer', 'canvas-zoom',
                         'model-keyword', 'openpose-editor', 'sd-dynamic-prompts', 'sd-webui-additional-networks',
                         'sd-webui-lora-block-weight', 'sd-webui-openpose-editor', 'sd-webui-prompt-all-in-one',
                         'stable-diffusion-webui-anti-burn', 'stable-diffusion-webui-state']

### Relevant log output

```shell
12:24:56-262019 WARNING  DirectML does not support torch Generator API.
12:24:56-265019 WARNING  diffusers_generator_device is automatically overriden to cpu.
12:24:56-266020 INFO     3 options are automatically overriden. If you want to keep them from overriding, run with
                         --experimental argument.
12:24:56-268019 INFO     Settings changed: 1 ['diffusers_generator_device']
```

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by Synitare** [comment]
For clarity, this option was stuck on CPU until I ran with --experimental, and generating images was pegging my CPU. Any attempt to change it back to default resulted in the error mentioned above. 

For some reason now after I attempted to recreate the issue by launching without --experimental, it is no longer producing the error. 

![image](https://github.com/vladmandic/automatic/assets/3077398/a73114bb-f656-4d09-819f-0e3058234dfa)
